### PR TITLE
Update release-notes.md for Feb 18 release

### PR DESCRIPTION
**Pull request by kevinlind**
Update release notes for Feb 18 release:
- Identity iOS v2.2.0

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -8,6 +8,24 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
...
+## February 18, 2020
+
+The following updates were made in this release:
+
+### iOS Identity 2.2.0
+
+* Report extension details to Mobile Core for improved logging and Griffon support.
+* Added APIs to support `completionHandler` callback that is available in iOS ACPCore version 2.5.0 on the following APIs:
+
+  * `appendToURL:witCompletionHandler`
```
Comment (emdobrin) [inline release-notes.md]: typo: withCompletionHandler

File: release-notes.md [hunk 2]
```diff
@@ -8,6 +8,24 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
...
+  * `getUrlVariablesWithCompletionHandler`
+  * `getIdentifiersWithCompletionHandler`
+  * `getExperienceCloudIdWithCompletionHandler`
+
+  When the `completionHandler` is used, and you are retrieving the Mobile SDK values, the timeout value is 500ms; if the operation times out or is not successful, an `NSError` is returned.
+
+Released with ACPCore version 2.5.0.
```
Comment (emdobrin) [inline release-notes.md]: Please also include these release notes in https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/blob/master/using-mobile-extensions/mobile-core/mobile-core-release-notes.md and update the iOS APIs documentation with the new methods

File: release-notes.md [hunk 3]
```diff
@@ -8,6 +8,24 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
...
 Please take a moment to fill out a [short survey](https://www.surveymonkey.com/r/AEPDocs) on how we can better assist you with enabling Adobe Experience Cloud solutions and services on your mobile apps.
 
+## February 18, 2020
+
+The following updates were made in this release:
+
+### iOS Identity 2.2.0
+
+* Report extension details to Mobile Core for improved logging and Griffon support.
+* Added APIs to support `completionHandler` callback that is available in iOS ACPCore version 2.5.0 on the following APIs:
```
Comment (rekrishn) [inline release-notes.md]: Please reword as follows:
"Added the following APIs to support the `completionHandler` callback that is available in iOS ACPCore version 2.5.0:"

File: release-notes.md (hunk 2 above)
Comment (kevinlind) [reply release-notes.md]: @emdobrin I'm updating the Identity API reference in another PR.

**Pull request merged**